controllers/observability/pod_disruption_budget_at_limit.go

```diff
@@ -0,0 +1,94 @@
+package observability
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/alertmanager"
+)
+
+var alertmanagerSvcHost = "alertmanager-main.openshift-monitoring.svc.cluster.local:9094"
+
+func (r *Reconciler) ensurePodDisruptionBudgetAtLimitIsSilenced() error {
+	if r.amApi == nil {
+		var err error
+		r.amApi, err = r.NewAlertmanagerApi()
+		if err != nil {
+			return fmt.Errorf("failed to initialize alertmanager api: %w", err)
+		}
+	}
+
+	amSilences, err := r.amApi.ListSilences()
+	if err != nil {
+		return fmt.Errorf("failed to list alertmanager silences: %w", err)
+	}
+
+	if FindPodDisruptionBudgetAtLimitSilence(amSilences) != nil {
+		log.Info("KubeVirt PodDisruptionBudgetAtLimit alerts are already silenced")
+		return nil
+	}
+
+	silence := alertmanager.Silence{
+		Comment:   "Silence KubeVirt PodDisruptionBudgetAtLimit alerts",
+		CreatedBy: "hyperconverged-cluster-operator",
+		EndsAt:    "3000-01-01T00:00:00Z",
+		Matchers: []alertmanager.Matcher{
+			{
+				IsEqual: true,
+				Name:    "alertname",
+				Value:   "PodDisruptionBudgetAtLimit",
+			},
+			{
+				IsRegex: true,
+				Name:    "poddisruptionbudget",
+				Value:   "kubevirt-disruption-budget-.*",
+			},
+		},
+		StartsAt: time.Now().Format(time.RFC3339),
+	}
+
+	if err := r.amApi.CreateSilence(silence); err != nil {
+		return fmt.Errorf("failed to create alertmanager silence: %w", err)
+	}
+	log.Info("Silenced PodDisruptionBudgetAtLimit alerts")
+
+	return nil
+}
+
+func (r *Reconciler) NewAlertmanagerApi() (*alertmanager.Api, error) {
+	httpClient := http.Client{}
+	httpClient.Transport = &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
please avoid `InsecureSkipVerify`

Reply:
i get a `tls: failed to verify certificate: x509: certificate signed by unknown authority`
do you know who i can avoid that?

Reply:
seems to be working using `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`